portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase

```diff
@@ -241,17 +239,15 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LRQA-61554."
+	@description = "This is a use case for LRQA-61554. Verify OAuth 2 works at the system level after upgrade from 7.2.10.1"
 	@priority = "5"
+	@uitest
```

Comment:
Here is how to use @uitest label rule:
Make sure that at least one Poshi test is still testing basic UI flows. Mark these so we don't convert them as we refactor the rest of our tests
Use @uitest as an annotation for this

So I think mark one instance level test and one system level test as uitest is enough.

I think we might need a discussion about how to use this @uitest label.
@gaborlovasdotcom

Reply:
Or not use UI test at all, if we don't benefit from this labeling.